ceph.spec.in

```diff
@@ -537,12 +537,14 @@ Requires:       python%{python3_pkgversion}-cherrypy
 Requires:       python%{python3_pkgversion}-jwt
 Requires:       python%{python3_pkgversion}-routes
 Requires:       python%{python3_pkgversion}-werkzeug
+Suggests:       python%{python3_pkgversion}-saml
```

Comment:
could you file a bugzilla ticket asking for a epel8 branch for python3-saml ? since we have fc33 branch. see https://src.fedoraproject.org/rpms/python3-saml. i believe we can have epel8 with minimum efforts.

Reply:
@ktdreyer and @bigjust are working on [releasing some pkgs from Ken's Ceph copr to EPEL](https://trello.com/b/ZAMLb0pT/ceph-el8).

Reply:
those packages are mostly the dependencies of python3-cherrypy , and they do not contain python-saml. see https://copr.fedorainfracloud.org/coprs/ktdreyer/ceph-el8/packages/

Reply:
Yup, I know. This whole thing comes from a chat Timothy A. Jeyasingh & I had yesterday with @ktdreyer at #ceph-dashboard Google Chat (that's why I prefer to keep discussions in shareable places :wink: ). Hope Ken doesn't mind I copy that here:

> In any regard, I think there may be no good reason why ceph.spec does not have Requires: python3-saml or Recommends: python3-saml
> It's quite likely an accident
> It is also important to verify that EPEL 8 has python3-saml in it
> Maybe that is why this was not done before
> I've been working with a newer member of our team (Justin Caratzas) to get all our dependencies in to EPEL 8
> I am thinking that python3-saml may not be on his list
> Justin is working from https://copr.fedorainfracloud.org/coprs/ktdreyer/ceph-el8/ , and tracking progress in https://trello.com/b/ZAMLb0pT/ceph-el8

Reply:
i see. and i still suggest file a bz for creating the epel8 branch of python-saml. so it can be at least be tracked. and won't be forgotten over time.

Reply:
Note that it was a deliberate decision to not add a dependency on python-saml to the spec file, as it's not required by default and the python code only requires the module when SSO is actively enabled.

Adding @ricardoasmarques to comment, as he's the original author of that feature.

Reply:
Having said that, I'm fine with adding it to the package dependencies in a "soft" form, e.g. as `Recommends`. Since the majority of users won't need this feature, I don't think we should complicate things unnecessarily.

Reply:
Thanks, @tchaikov & @LenzGr for your feedback!

@tchaikov there's already this [Bugzilla](https://bugzilla.redhat.com/show_bug.cgi?id=1819302) (Ken and Timothy are working on that).

Reply:
@tchaikov please let me know what you exactly mean, cause I don't follow. From my understanding, @ktdreyer & @bigjust were undertaking the addition of several Ceph deps. to EPEL8 (including this one and their deps) which will in turn involve the BZs to Fedora EPEL. What am I missing there? Thanks!

Reply:
discussed with @epuertat offline, my meant to suggest we need to track the efforts to package python3-saml in EPEL8, so our user can at least expect python3-saml when the package is found in the (weak) dependency list of ceph.

but since we are fully aware of this and we are going to package it for EPEL8, i am fine now =) 

thanks!

Reply:
Who is packaging this for EPEL 8?

Reply:
i think Justin is on this. @epuertat, please correct me if i am wrong.

Reply:
> Who is packaging this for EPEL 8?

@ktdreyer I initially understood @bigjust and you we porting stuff to EPEL8, but after our last discussion, Timothy agreed to take over on this.